alpha/apps/kaltura/modules/extwidget/actions/playManifestAction.class.php

```diff
@@ -114,6 +114,11 @@ class playManifestAction extends kalturaAction
 	 * @var DeliveryProfileDynamicAttributes
 	 */
 	private $deliveryAttributes = null;
+
+	/**
+	 * @var int
+	 */
+	private $deliveryProfileId = null;
```

Comment:
Where do you use this private member except when doing:
$this->deliveryProfileId = $this->getRequestParameter( "deliveryProfileId", null );
$this->deliveryAttributes->setDeliveryProfileId($this->deliveryProfileId);

If its no used anywhere else than remove it and simply do:
$this->deliveryAttributes->setDeliveryProfileId($this->getRequestParameter( "deliveryProfileId", null ););